Sources/Engine/Entities/Entity.h

```diff
@@ -694,18 +694,16 @@ inline CEntity& CEntityPointer::operator*(void) const { return *ep_pen; }
 /////////////////////////////////////////////////////////////////////
 // Reference counting functions
 inline void CEntity::AddReference(void) { 
-  if (this!=NULL) {
-    ASSERT(en_ctReferences>=0);
-    en_ctReferences++; 
-  }
+  ASSERT(this!=NULL);
```

Comment:
> In that case, something went terribly wrong. Do you know what called AddReference on a NULL-pointer?

I don't; maybe it was something that was fixed later (or it was a run that failed to load the Entities DLL or something...?).  If it's not actually a problem, I'm happy.  :)